packages/holocron/__tests__/ducks/load.spec.js

```diff
@@ -277,6 +278,31 @@ describe('loadModule', () => {
     expect(rebuildReducer).toHaveBeenCalled();
   });
 
+  it('should not rebuild the store when a module is loaded successfully does not have a reducer', async () => {
```

Comment:
```suggestion
  it('should not rebuild the store when a module without a reducer is loaded successfully', async () => {
```
This could be clearer.